### ISSUE TITLE
Request: Add collected pets breed to the tooltip similar to how Pettracker does it

### ISSUE THREAD

**Issue by jhammersholt** [GurliGebis/WoWAddon-BattlePetCompletionist#46]
I would absolutely love the ability to see which pets i already have in the tooltip of pets (journal, cages or unlearned item) similar to how pettracker does it, with one minor change

Instead of showing the icon for breed i would love to choose between icon and breed text (P/P, H/S, S/S and so forth)

![image](https://github.com/user-attachments/assets/c35e082f-86e1-4e60-9769-f15a7160e7e1)

**Comment by jhammersholt** [comment]
And i just realized Rematch needs some attention for that to work as well (i guess?)

![image](https://github.com/user-attachments/assets/5ca11059-6429-49e3-8f6e-171967960c4f)

**Comment by davidmc24** [comment]
I agree that some additional modes and/or customization will make sense at some point. I tend to collect max rare, without paying too much attention to breeds, so what I would tend towards would be the level of each pet, colored by rarity.

**Comment by JPEscher** [comment]
Definitely a Nice-to-Have Feature.

**Comment by GurliGebis** [comment]
Looking at the API for getting pet info, I'm not really sure how to get the needed info: https://wowpedia.fandom.com/wiki/API_C_PetJournal.GetPetInfoByPetID

**Comment by JPEscher** [comment]
@GurliGebis i believe the group behind the api stuff have moved a few times and the current is https://warcraft.wiki.gg/
that said aside form some naming it looks like there's no change from that link. however this one might be useful for this:  https://warcraft.wiki.gg/wiki/API:C_PetJournal.GetPetStats

that being said i think this might be covered by battle pet breed id. 
also rematch seems to be dead so i moved to petmatch. not sure if its petmatch or battlepet breedid but i see

<img width="359" height="500" alt="Image" src="https://github.com/user-attachments/assets/0adcba49-cb63-4ced-934c-2b2f276480c9" />

<img width="683" height="492" alt="Image" src="https://github.com/user-attachments/assets/ebb13daf-716d-4909-a632-58ba81ca2469" />

so something i have is showing that... 
testing with battlepet breedid on and petmatch off i dont get the breed display.
testing with battlepet breedid off and petmatch on i dont get the breed display.
so i think petmatch is displaying when it can interact with battlepet breedid

that might be the way to go, to integrate with and run off battlepet breedid

if that's desired, here's the link to BPBID's source so you can contact the author. https://github.com/MMOSimca/BattlePetBreedID
or on curse at https://www.curseforge.com/wow/addons/battle_pet_breedid
hope this helps.

if you'd like to talk to Fwen, the PetMatch Author, she is on warcraftpets & xu-fu discord and the mod is at https://www.curseforge.com/wow/addons/petmatch

**Comment by JPEscher** [comment]
ooh i missed this si for the tookl tip as well.
i see 

<img width="312" height="378" alt="Image" src="https://github.com/user-attachments/assets/6fe7f43f-0a1b-43dd-883a-73cc60b9d1b3" />

again, testing with battlepet breedid on and petmatch off i do get the breed display.
testing with battlepet breedid off and petmatch on i dont get the breed display. 

so im pretty sure battlepet breedid already adds it to the tool tip.

**Comment by GurliGebis** [comment]
@JPEscher any ideas on how to convert C_PetJournal.GetPetStats values into breed names? (B/B, H/P etc.)

**Comment by JPEscher** [comment]
not yet. And having noticed BPBID does this already I wonder if leveraging them as an integration might be better.

**Comment by JPEscher** [comment]
just noticed this. https://github.com/MMOSimca/BattlePetBreedID/issues/10 which oddly mentions battle pet count ng

**Comment by GurliGebis** [comment]
I see - and I see he does it be having a huge array with this.
I like the idea, but I'll close this ticket for now, since I don't see it happening.

If we can come up with a solution that is low on maintenance, sure, we can reopen it.
